### PR TITLE
Files cleanup

### PR DESCRIPTION
**Pull request by freisei**

### CODE REVIEW

**Comment by kiall** [comment]
Looks like:

* psng/icons_1024x768/probe_screen.glade should go to psng/psng_1024x768.glade + have the icon paths updated.
* psng/psng_scroll.glade still has the old icons paths.
* psng/icons_1024x768/probe_screen.hal should be deleted (no need for 2 copies of this file)

File: psng/dev_environment_branch.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+in your config dir execute:
+
+  git init
+  git remote add origin git@github.com:linuxcnc-probe-screen/probe-screen-ng.git
+  git fetch
+  git branch files_cleanup origin/files_cleanup
+  git checkout files_cleanup
```
Comment (kiall) [inline psng/dev_environment_branch.md:7]: Better yet, you can do this:


    cd $HOME
    git clone git@github.com:linuxcnc-probe-screen/probe-screen-ng.git
    cd $HOME/linuxccnc/configs/MyConfigName
    ln -s $HOME/probe-screen-ng/macros macros-psng
    ln -s $HOME/probe-screen-ng/psng psng
    ln -s $HOME/probe-screen-ng/python python

This way, you have a clean clone of the repository without all your extra linuxcnc files showing up in `git status` etc, and can still easily make a change, then just restart linuxcnc to try it out.

**Comment by freisei** [comment]
Wohooow. PRs are fine things

**Review by kiall — APPROVED** [review]
Looks good to me! If you're happy, lets merge it :)

**Comment by freisei** [comment]
Jippiiiiie! My first PR will be merged!

**Pull request merged**